### ISSUE TITLE
osx_bundle: jhbuild can't download autoconf

### ISSUE THREAD

**Issue by smammy** [quodlibet/quodlibet#3278]
Steps to reproduce
------------------
1. `git clone https://github.com/quodlibet/quodlibet.git`
2. `cd quodlibet/osx_bundle`
3. `./bootstrap.sh`
4. `./build.sh`

Expected Output
---------------
Build should complete without error.

Actual Output
-------------
Partway through the build, I see:
```
*** Checking out autoconf-archive *** [30/104]
wget --continue https://mirror.reismil.ch/gnu/autoconf-archive/autoconf-archive-2018.03.13.tar.xz -O /Users/sam/Files/Projects/Music/quodlibet/osx_bundle/_home/jhbuild_checkoutroot/pkgs/autoconf-archive-2018.03.13.tar.xz
--2019-11-20 10:59:12--  https://mirror.reismil.ch/gnu/autoconf-archive/autoconf-archive-2018.03.13.tar.xz
Resolving mirror.reismil.ch (mirror.reismil.ch)... 37.252.190.140
Connecting to mirror.reismil.ch (mirror.reismil.ch)|37.252.190.140|:443... connected.
Unable to establish SSL connection.
*** Error during phase checkout of autoconf-archive: ########## Error running wget --continue https://mirror.reismil.ch/gnu/autoconf-archive/autoconf-archive-2018.03.13.tar.xz -O /Users/sam/Files/Projects/Music/quodlibet/osx_bundle/_home/jhbuild_checkoutroot/pkgs/autoconf-archive-2018.03.13.tar.xz *** [30/104]
```

Test System
-----------

### Which version of Quod Libet?
`master`

### Which operating system 
macOS 10.14.6

### If it's audio-related, what back-end?
n/a

Additional Information
----------------------
This is the fix, which I will eventually submit as a PR.
```
diff --git a/osx_bundle/modulesets/quodlibet.modules b/osx_bundle/modulesets/quodlibet.modules
index 17c79498c..264d87259 100644
--- a/osx_bundle/modulesets/quodlibet.modules
+++ b/osx_bundle/modulesets/quodlibet.modules
@@ -74,7 +74,7 @@
   <repository type="tarball" name="nasm"
               href="http://www.nasm.us/pub/nasm/releasebuilds/"/>
   <repository type="tarball" name="autoconf-archive"
-              href="https://mirror.reismil.ch/gnu/autoconf-archive/"/>
+              href="https://ftpmirror.gnu.org/autoconf-archive/"/>

   <metamodule id="meta-bootstrap">
     <dependencies>
```

**Comment by lazka** [comment]
> This is the fix, which I will eventually submit as a PR.

sounds good!

**Comment by lazka** [comment]
Fixed in 8a4752a3dfd2f26171ec
